pkg/config/load.go

```diff
@@ -33,6 +33,21 @@ type Prowgen struct {
 	Expose bool `json:"expose,omitempty"`
 	// Rehearsals declares any disabled rehearsals for jobs
 	Rehearsals Rehearsals `json:"rehearsals,omitempty"`
+	// Set which cluster and architecture should the images be promoted from
+	Images Images `json:"postsubmit_images"`
+}
+
+type Images struct {
+	// amd64 is the default arch, falsify this field to skip it
+	SkipDefaultArch bool   `json:"skip_default_arch,omitempty"`
```

Comment:
I don't think we need this. We will always build for amd64. We only need to allow users to specify the extra architectures